ui/components/snackBar/topRightSnackBar.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react'
```

Comment:
Let's abstract this component to be any position snack bar, receiving message, and position props.